### PR TITLE
fix med centers admin

### PR DESCRIPTION
**Pull request by KLIM-nk01**

### CODE REVIEW

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  const dispatch = useDispatch();
+  const { medCenters } = useTypesSelector((state) => state.medCenter);
+  console.log(medCenters);
+  useEffect(() => {
+    dispatch(fetchMedCenters());
+  }, []);
+  return (
+    <MedCentersListWrapper>
+      {medCenters.map((medCenter) => (
+        <CenterWrapper>
```
Comment (katsiaryna-savchuk-itechart) [inline src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx:19]: key prop is missed

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { useTypesSelector } from '@hooks/UseTypedSelector';
+import { fetchMedCenters } from '@store/actionCreators/medCenters';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { MedCentersListWrapper, CenterWrapper } from './MedCentersStyle';
+
+const MedCentersList: React.FC = () => {
+  const dispatch = useDispatch();
+  const { medCenters } = useTypesSelector((state) => state.medCenter);
+  console.log(medCenters);
```
Comment (katsiaryna-savchuk-itechart) [inline src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx:12]: remove log

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  useEffect(() => {
+    dispatch(fetchMedCenters());
+  }, []);
+  return (
```
Comment (katsiaryna-savchuk-itechart) [inline src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx:16]: pls add lines in components for better readability

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import Button from '@components/common/Button/Button';
+import MedCenter from '@components/MedCentersPage/MedCentersList/MedCenter/MedCenter';
+import { useTypesSelector } from '@hooks/UseTypedSelector';
+import { fetchMedCenters } from '@store/actionCreators/medCenters';
+import React, { useEffect } from 'react';
```
Comment (katsiaryna-savchuk-itechart) [inline src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx:5]: react usually goes the first in the imports

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCenterAddForm/MedCenterAddForm.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+const MedCenterForm = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+  return (
+    <MedCenterFormWrapper>
+      <FormName>Edit or create new medical center</FormName>
+      <Input primary label="Email" name="email" register={register('email')} errors={errors} />
```
Comment (katsiaryna-savchuk-itechart) [inline src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCenterAddForm/MedCenterAddForm.tsx:16]: is there gonna be only one field in the form for centers?

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx (hunk 1 above)
Comment (KLIM-nk01) [reply src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx]: https://github.com/KLIM-nk01/virtual-ambulance/pull/26/commits/6a7aa338bd2cf39a43f0bb25411635179e962756#diff-23c5f3b8e346a5dc5889a2295907f46b739175af80e238e38c625ccb47056ed7

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx (hunk 2 above)
Comment (KLIM-nk01) [reply src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx]: https://github.com/KLIM-nk01/virtual-ambulance/pull/26/commits/6a7aa338bd2cf39a43f0bb25411635179e962756#diff-23c5f3b8e346a5dc5889a2295907f46b739175af80e238e38c625ccb47056ed7

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx (hunk 3 above)
Comment (KLIM-nk01) [reply src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx]: https://github.com/KLIM-nk01/virtual-ambulance/pull/26/commits/6a7aa338bd2cf39a43f0bb25411635179e962756#diff-23c5f3b8e346a5dc5889a2295907f46b739175af80e238e38c625ccb47056ed7

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx (hunk 4 above)
Comment (KLIM-nk01) [reply src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCentersList.tsx]: https://github.com/KLIM-nk01/virtual-ambulance/pull/26/commits/6a7aa338bd2cf39a43f0bb25411635179e962756#diff-23c5f3b8e346a5dc5889a2295907f46b739175af80e238e38c625ccb47056ed7

File: src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCenterAddForm/MedCenterAddForm.tsx (hunk 5 above)
Comment (KLIM-nk01) [reply src/components/AdminPanel/AdminWorkSpace/AdminMedCenters/MedCenters/MedCenterAddForm/MedCenterAddForm.tsx]: https://github.com/KLIM-nk01/virtual-ambulance/pull/26/commits/6a7aa338bd2cf39a43f0bb25411635179e962756#diff-82c3df5933f2e8057602e5ffc8dca1781e551a9ea622c252f31d3ac117e2fe1f

**Pull request merged**